qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -93,6 +132,270 @@ def __init__(self, control_config: Dict[Tuple[int, ...], List[ControlChannel]] =
         self._hash_to_counter_map = {}
         self._parameter_counter = 0
 
+        self._library = None
+        if library is not None:
+            self._library = library
+
+            # Add the basis gates
+            for gate in library.basis_gates:
+                self.add_schedule(library[gate], num_qubits=library.num_qubits(gate))
+
+            # Add the default values
+            if add_parameter_defaults:
+                for param_conf in library.default_values():
```

Comment:
param_conf can be named tuple? It seems like very sensitive to the update of default_values method, since the library subclass can implement arbitrary code (we can guarantee number of args matches with the`self.add_parameter_value` with typecheck).

Reply:
How about this: 0dabeed